staticfiles/js/worker/services/worker.service.js

```diff
@@ -29,16 +29,16 @@
         url: 'http://share-quick.com/cr/getWorkerProfile.php',
         method: 'POST'
       });
-      
-    }   
+
+    }
 
     function getWorkerTaskPortfolio() {
       return $http({
-        url: 'https://api.myjson.com/bins/q7jc',
+        url: '/api/worker/1/portfolio/',
         method: 'GET'
       });
```

Comment:
the response code 304 was for static files, not for api request, sorry for confusion. But the problem still exist, I think HttpService is not returning response.